game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModel.java

```diff
@@ -154,19 +138,11 @@ public Object getValueAt(final int rowIndex, final int columnIndex) {
       case Comments:
         return description.getComment();
       case Started:
-        return formatBotStartTime(description.getStartDateTime());
```

Comment:
Interesting choice.
I believe I was the last person to touch this method and for me it seemed perfectly reasonable that the "UI" part gets to decide how the Instant is formatted.

If you insist this is the right thing to do, then the same should apply to `description.isPassworded()` as well as `description.getRound()` IMO, because in those cases the description only supplies the raw data, which is then formatted by this class.

Reply:
Good call-out.  It did look like a feature envy code smell having the method in 'tableModel'. Passing a formatter object to 'description' could be a way to god too, I suspect that is over-engineered though.

Game table model though has too many responsibilities as is. It's basically a UI class, IMO should be focused mostly on layout and satisfying the swing table contract.